opencti-platform/opencti-graphql/src/migrations/1712129378000-investigations-and-dashboards-capabilities.js

```diff
@@ -0,0 +1,30 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { addCapability } from '../domain/grant';
+import { elLoadById, elReplace } from '../database/engine';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  // ------ Add Manage Public Dashboards capability
+  await addCapability(context, SYSTEM_USER, {
+    name: 'EXPLORE_EXUPDATE_PUBLISH',
```

Comment:
this capability has already been created in existing platforms, so I think we should rename it like the others.

Reply:
migration had not been played